src/style.css

```diff
@@ -0,0 +1,90 @@
+* {
+  margin: 0;
```

Comment:
if you want to reset margins of body, use it with body tag, no need to use *-selector here